drivers/modem/ublox-sara-r4.c

```diff
@@ -41,16 +41,16 @@ enum mdm_control_pins {
 static struct modem_pin modem_pins[] = {
 	/* MDM_POWER */
 	MODEM_PIN(DT_INST_0_UBLOX_SARA_R4_MDM_POWER_GPIOS_CONTROLLER,
-		  DT_INST_0_UBLOX_SARA_R4_MDM_POWER_GPIOS_PIN, GPIO_DIR_OUT),
+		  DT_INST_0_UBLOX_SARA_R4_MDM_POWER_GPIOS_PIN, GPIO_OUTPUT),
```

Comment:
One change we could make here is to take into account the GPIO flags:
```
	MODEM_PIN(DT_INST_0_UBLOX_SARA_R4_MDM_POWER_GPIOS_CONTROLLER,
		  DT_INST_0_UBLOX_SARA_R4_MDM_POWER_GPIOS_PIN,
		  DT_INST_0_UBLOX_SARA_R4_MDM_POWER_GPIOS_FLAGS | GPIO_OUTPUT),
```

Reply:
This would be similar to how we changed the WNC driver.

Reply:
Yes, this makes sense. Let me make this change